documentation/modules/snip-migrating-luns.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: SNIPPET
+
+{project-short} supports direct migration of virtual machines with logical unit numbers (LUNs), other than Fibre Channel, from {rhv-full} ({rhv-short}) source providers as follows:
```

Comment:
it's not clear to me what "direct migration" means, how about dropping the "direct" part?
```suggestion
{project-short} supports migration of virtual machines with logical unit numbers (LUNs), other than Fibre Channel, from {rhv-full} ({rhv-short}) source providers as follows:
```